src/Field/Connection/Link.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_tpr\Field\Connection;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Url;
+
+/**
+ * Provides a domain object for TPR connection type of LINK.
+ */
+final class Link extends Connection {
```

Comment:
This can be changed to extend OpeningHourBase class since this has same fields. The base class should probably be renamed to something else tho.

Reply:
Good idea. Changed the class to `TextWithLinkBase` as it was best name I came up with.